APMrover2/system.cpp

```diff
@@ -369,3 +369,21 @@ bool Rover::is_boat() const
 {
     return ((enum frame_class)g2.frame_class.get() == FRAME_BOAT);
 }
+
+// position_ok - returns true if the horizontal absolute position is ok and home position is set
+bool Rover::position_ok(nav_filter_status& filt_status)
```

Comment:
I think the method name is "is_position_ok".
return status , ok is true, fail is false